logstash-filter-date.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-filter-date'
-  s.version         = '3.0.3'
+  s.version         = '3.0.4'
```

Comment:
Hmm, thoughts on making it 3.1.0? IMO, performance improvement _is_ a feature that will be visible to users.